doc/manual/configuration.rst

```diff
@@ -1742,9 +1742,12 @@ are applied after string substitution. The general syntax looks like the followi
           url:
             pattern: "foo"
             replacement: "bar"
+        if: !expr |
+          "${USE_OVERRIDES} == 1"
```

Comment:
I think the condition is wrong: it is just a single string that is interpreted as `True`. Maybe it would be nice to have a more useful example too. Something like:

```suggestion
          "${BOB_RECIPE_NAME}" == "foo"
```